multihist_plotter.C

```diff
@@ -0,0 +1,180 @@
+
+#include "tdrstyle.C"
+#include "TH1.h"
+#include "TH1F.h"
+
+int dolog=1;
+void multihist_plotter() 
+{ 
+    // create vector with signal histogram names
+    // add push_back lines to input the names of the histograms
+    // you're interested in plotting from the signal samples
+    std::vector<char*> sighnames;
+    sighnames.push_back("bjet_alphamax");
+    sighnames.push_back("darkjet_alphamax");
+    // create vector with background histogram names
+    // works same as the signal vector, above
+    std::vector<char*> bkghnames;
+    bkghnames.push_back("jet_alphamax");
+
+    char* atitle = "jet_alphamax";
+     
+    // lists of colors for the different histograms
+    int sig_colors[4] = {2,3,4,5};
+    int bkg_colors[4] = {6,7,8,9};
+ 
+    // option to scale to cross section (rather than just 1)
+    // along with xs and lumi values to use to do so
+    int scaletoxs = 0;
+    float ttbarxs = 888000.; // in fb
+    float darkxs = 18.45402; // in fb
+    float lumi = 100.; // fb^-1
+
+    TFile *f1 = new TFile("results_signal.root");
+    TFile *f2 = new TFile("results_ttbar.root");  
+
+
+    gStyle->SetOptStat(0);
+
+    TString canvName = "Fig_";
+    canvName += atitle;
+
+
+    int W = 800;
+    int H = 600;
+    TCanvas* canv = new TCanvas(canvName,canvName,50,50,W,H);
+    // references for T, B, L, R
+    float T = 0.08*H;
+    float B = 0.12*H; 
+    float L = 0.12*W;
+    float R = 0.04*W;
+
+    //canv = new TCanvas(canvName,canvName,50,50,W,H);
+    canv->SetFillColor(0);
+    canv->SetBorderMode(0);
+    canv->SetFrameFillStyle(0);
+    canv->SetFrameBorderMode(0);
+    //canv->SetLeftMargin( L/W );
```

Comment:
probably need to revisit these - the x-axis title on your example plot seems to be cut off (and the y-axis title is missing completely).

Reply:
Great! Can you check the plot margins (see below comment)? Then this will be ready to go in

Reply:
Managed to fix this by toying around with the axis title offsets until they became visible again. I've attached an example.
![fig_jet_alphamax_log](https://user-images.githubusercontent.com/13453984/30129712-d764a696-9314-11e7-8a0c-3fb45c2b5a81.png)